.github/workflows/lint-files.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     timeout-minutes: 20
     steps:
       - name: Harden Runner # Security agent: blocks outbound traffic & detects code overwrite to prevent breaches
-        uses: step-security/harden-runner@128a63446a954579617e875aaab7d2978154e969
+        uses: step-security/harden-runner@17d0e2bd7d51742c71671bd19fa12bdc9d40a3d6
```

Comment:
This is already in develop. Lets merge that into this and remove this unrelated diff.